proxy/http/HttpTransact.cc

```diff
@@ -3460,7 +3460,7 @@ HttpTransact::HandleStatPage(State *s)
       s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, s->internal_msg_buffer_type, len);
     }
   } else {
-    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 9);
+    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 10);
```

Comment:
I agree, but this is apparently a very common pattern (the magic numbers all over)-- so I fixed the number in an attempt to be consistent. I'm fine to change it though-- it just won't match :/ Maybe we should make another ticket for cleaning these sorts of things up? Is there a linter that can check for magic numbers? (so we can make an automated check for them)